### PR TITLE
Final scraper

### PR DESCRIPTION
**Pull request by squush**
May need to still embed this scraping code elsewhere - right now the index page takes about 5 seconds to load. May need to db:migrate if you haven't already.

### CODE REVIEW

File: app/policies/tournament_policy.rb [hunk 1]
```diff
@@ -4,5 +4,9 @@ class Scope < Scope
     def resolve
       scope.all
     end
+
+    def index
```
Comment (andygw) [inline app/policies/tournament_policy.rb:8]: Was this needed after all? I thought it worked with just the policy_scope thing you had in the tournament controller

File: scrape.rb [hunk 2]
```diff
@@ -14,9 +14,9 @@
       location = tournament.css('a').children.text
       # May want to use this date method below to get the tournament's start date
       # So that we can sort upcoming events by start date
-      # month_and_day = tournament.css('td')[-2].children.text[/^\d*\/\d*/]
-      # year = tournament.css('td')[-2].children.text[/\d\d\d\d/] || Date.today.year
-      # sortable_date = Date.parse("#{year}/#{month_and_day}")
+      month_and_day = tournament.css('td')[-2].children.text[/^\d*\/\d*/]
```
Comment (andygw) [inline scrape.rb:17]: If scrape.rb is no longer needed, we should delete it from the repo

File: app/policies/tournament_policy.rb (hunk 1 above)
Comment (squush) [reply app/policies/tournament_policy.rb:8]: I was told yes... but not positive.

File: scrape.rb (hunk 2 above)
Comment (squush) [reply scrape.rb:17]: I think it would be useful to leave it in, just cause I will need to test more scraping to get all players for a tournament

**Pull request merged**